### PR TITLE
feat(store): Only add metrics deliverables to file store

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR adds | fixes:
- Only add metric deliverables as output file for downstream use 

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [x] Installation, unit and integration test suite pass

### Review:
- [x] Code review
- [x] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [x] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
To be used with: https://github.com/Clinical-Genomics/cg/pull/1245

**Comment by henrikstranneheim** [comment]
MIP run started via cg:
Screenshot 2021-08-08 at 21 59 22

Only one qc_collect_ar record:
```
[31|1898|1898] θ70° 5d [hiseq.clinical@hasta:~] [S_main] $ grep qccollect_ar -A 2 -B 4 /home/proj/stage/rare-disease/cases/justhusky/analysis/justhusky_deliverables.yaml
  - format: meta
    id: justhusky
    path: /home/proj/stage/rare-disease/cases/justhusky/analysis/justhusky_metrics_deliverables.yaml
    path_index: ~
    step: qccollect_ar
    tag: ~
  - format: meta
[31|1898|1898] θ72° 5d [hiseq.clinical@hasta:~] [S_main] $
```

File: lib/MIP/Recipes/Analysis/Mip_qccollect.pm [hunk 1]
```diff
@@ -201,16 +201,6 @@ sub analysis_mip_qccollect {
...
-            {
-                format           => q{meta},
-                id               => $case_id,
-                path             => $outfile_path,
-                recipe_name      => $recipe_name,
-                sample_info_href => $sample_info_href,
-            }
-        );
-
         if ( defined $active_parameter_href->{qccollect_store_metrics_outfile} ) {
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Mip_qccollect.pm:204]: Do we need to keep this condition? Don't we always wanna store this file?

**Review by jemten — COMMENTED** [review]
Looks good! 
Since this is a PR to master we should update version number and update the changelog as well.

File: lib/MIP/Recipes/Analysis/Mip_qccollect.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Mip_qccollect.pm:204]: Yes, I think you are right.

Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Mip_qccollect.pm]: fixed

**Pull request merged**